zerver/models.py

```diff
@@ -768,6 +768,17 @@ def to_dict_cache_key(message, apply_markdown):
     # type: (Message, bool) -> text_type
     return to_dict_cache_key_id(message.id, apply_markdown)
 
+class Topic(ModelReprMixin, models.Model):
```

Comment:
Maybe add a comment here explaining a little bit the story of this table (primarily that it is not in production use yet), to help avoid confusing people looking through the code?